src/components/ProfileView.jsx

```diff
@@ -34,7 +34,10 @@ const ProfileView = ({ t, fields, passphrase, isFetching, onFieldChange, onPassp
         name='locale'
         label={t('ProfileView.locale.title')}
         description={t(`ProfileView.locale.label`)}
-        options={LANG_OPTIONS.map(lang => t(`ProfileView.locale.${lang}.text`))}
+        options={LANG_OPTIONS.map(lang => [{
```

Comment:
As `Array.prototype.map` already return an array, you don't have to wrap your return object in an array.
```javascript
LANG_OPTIOS.map(lang => { return {
    value: lang,
    text: t(`ProfileView.locale.${lang}.text`)
}}
````

is enough.